wasp/drivers/battery.py

```diff
@@ -79,6 +89,5 @@ def level(self):
         :returns: Estimate battery level in percent.
         """
         mv = self.voltage_mv()
-        level = ((19 * mv) // 100) - 660
-        level = min(100, max(0, level))
-        return level
+        level = int((mv - 3500) / (700) * 100)  # 0.7V is 4.2-3.5V
```

Comment:
Does this fix #338?

Reply:
Not sure but it assumes the level goes from 3.5 to 4.2V. Obviously the level value will not evolve linearly in that case. Personaly I'm using the voltage value directly to know how much power I have left